taprpc/tapchannelrpc/tapchannel.proto

```diff
@@ -121,6 +121,11 @@ message SendPaymentRequest {
     // contain a valid keysend record (key 5482373484 and a 32-byte preimage
     // that corresponds to the payment hash).
     routerrpc.SendPaymentRequest payment_request = 4;
+
+    // The rfq scid to use for this payment. If the user sets this value then
+    // the payment will immediately be dispatched, skipping the rfq negotiation
+    // phase, and using the following rfq scid.
+    uint64 scid = 5;
```

Comment:
Should this skip RFQ completely or just override scid? Where does the asset rate come from?

Reply:
I think we'll instead want the full 32-byte RFQ ID to be passed in. Then we can validate it exists and hasn't expired yet. If all is okay, we can easily derive the SCID from it.

Reply:
I don't really see the need to pass the full `[32]byte` rfq id?

We'll still have to perform the lookup with `r.cfg.RfqManager.PeerAcceptedSellQuotes()` which uses the scid as the key

Also, if an entry exists in that list it means that:
- Quote exists
- Quote hasn't expired

Reply:
I would argue that the RFQ ID provides a more general and accurate way to reference an RFQ quote. The SCID is downstream of the RFQ quote and serves as a specific property of it. Using the RFQ ID in the interface ensures clarity and reinforces that we are referring to the entire quote (including the asset rate) rather than just the SCID.